src/webview/static/base.css

```diff
@@ -564,3 +564,71 @@ hr {
   border: 1px dashed hsla(0, 0%, 50%, 0.5);
   border-radius: 0.5rem;
 }
+.spoiler-block {
+  border: 	hsla(0, 0%, 50%,1) 1px solid;
+  padding: 2px 8px 2px 10px;
+  border-radius: 10px;
+  position: relative;
+  top: 1px;
+  display: block;
+  margin: 5px 0 15px 0;
+}
+.spoiler-block .spoiler-header {
+  padding: 5px;
+  font-weight: bold;
+}
+.spoiler-block .spoiler-content {
+  overflow: hidden;
+  border-top: hsla(0, 0%, 50%,1) 0px solid;
+  transition: height 0.4s ease-in-out, border-top 0.4s step-end, padding 0.4s step-end;
+  padding: 0px;
+  height: 0px;
+}
+.spoiler-block .spoiler-content.spoiler-content-open {
+  border-top: hsla(0, 0%, 50%,1) 1px solid;
+  transition: height 0.4s ease-in-out, border-top 0.4s step-start, padding 0.4s step-start;
+  padding: 5px;
+  height: auto;
+}
+.spoiler-block .spoiler-button {
```

Comment:
In the webapp's version, there's one more rule here, involving hover:
```scss
        .spoiler-button {
            float: right;
            width: 25px;
            height: 25px;
            &:hover .spoiler-arrow {
                &::before,
                &::after {
                    background-color: hsl(0, 0%, 50%);
                }
            }
        }
```
The commit message says that was dropped because hover doesn't exist for us... but I'm not 100% sure that's true, given that people can use alternate inputs with mobile devices, and often do for accessibility reasons. So let's include that rule too -- it'll make these styles exactly match the webapp's version (modulo my small other comments), which helps simplify thinking about them and also maintaining them in the future.